webportal/src/app/layout/layout.component.ejs

```diff
@@ -114,6 +114,6 @@
     <div class="control-sidebar-bg"></div>
 
   </div>
-
+  <script src="scripts/env.js"></script>
```

Comment:
Why do you add this to this .ejs file? I'm wondering whether 'require' the .js file in job-view.component.js is sufficient?

Reply:
@hwuu `layout.component.ejs` is the base template of all html pages, it makes all page scripts can access `window.ENV` by `require('webportal.config.js')`